src/views/2_SessionComponent.tsx

```diff
@@ -25,6 +25,10 @@ export class SessionComponent extends React.Component<Props, {}> {
     }
 
     componentDidMount() {
+        this.session.onscroll = () => {
+            const shutterOffset = this.session.scrollTop;
+            this.shutter.style.transform = `translateY(${shutterOffset}px)`;
+        };
```

Comment:
```
<div>
  <shutter/>
  <scrollable/>
</div/
```

If you have a structure like this, it wouldn't need to be scrolled at all. As far as I remember, it means we have to move the shutter one level up.

By the way, is there a better word for this thing than "shutter"?